gef-ui/package.json

```diff
@@ -72,7 +73,8 @@
     "path": "0.12.7",
     "redis": "^3.1.2",
     "sanitize-html": "^2.11.0",
-    "validator": "^13.9.0"
+    "supertest": "^6.3.3",
```

Comment:
`supertest` should be a `devDependency` we only use it for tests

Reply:
Lint error: https://github.com/UK-Export-Finance/dtfs2/actions/runs/5793973502/job/15702680232?pr=1941